alonzo/impl/test/lib/Test/Cardano/Ledger/Alonzo/Serialisation/Generators.hs

```diff
@@ -45,9 +46,10 @@ instance Arbitrary ExUnits where
   arbitrary = ExUnits <$> arbitrary <*> arbitrary
 
 instance
-  ( Era era,
-    Mock (Crypto era),
+  ( ShelleyBased era,
```

Comment:
We're trying to deprecate the various `ShelleyBased` constraints, so you should use the various `UseFoo` constraints as needed.